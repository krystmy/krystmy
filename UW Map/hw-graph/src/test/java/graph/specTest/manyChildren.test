# Test printing out multiple children of a node

# Create a graph
CreateGraph graph1

# Add some nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3
AddNode graph1 n4
AddNode graph1 n5

# Add an edge
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n3 e2
AddEdge graph1 n1 n4 e3
AddEdge graph1 n1 n5 e4

# Print the nodes in the graph and the children of n1
ListNodes graph1
ListChildren graph1 n1