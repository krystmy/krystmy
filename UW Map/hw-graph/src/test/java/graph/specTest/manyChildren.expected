# Test printing out multiple children of a node

# Create a graph
created graph graph1

# Add some nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1
added node n4 to graph1
added node n5 to graph1

# Add an edge
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n3 in graph1
added edge e3 from n1 to n4 in graph1
added edge e4 from n1 to n5 in graph1

# Print the nodes in the graph and the children of n1
graph1 contains: n1 n2 n3 n4 n5
the children of n1 in graph1 are: n2(e1) n3(e2) n4(e3) n5(e4)